include/openthread/platform/crypto.h

```diff
@@ -170,8 +170,9 @@ typedef struct otPlatCryptoSha256Hash otPlatCryptoSha256Hash;
  */
 typedef struct otPlatCryptoEcdsaKeyPair
 {
-    uint8_t mDerBytes[OT_CRYPTO_ECDSA_MAX_DER_SIZE];
-    uint8_t mDerLength;
+    uint32_t mKeyRef;
+    uint8_t *mDerBytes;
+    uint8_t  mDerLength;
```

Comment:
Add documentation on how/when `mKeyRef` is used?

Reply:
Will add documentation in next push.